midp/background.js

```diff
@@ -66,7 +66,7 @@ Native["com/nokia/mid/s40/bg/BGUtils.waitUserInteraction.()V"] = function() {
       localmsgServerWait = resolve;
     });
   }).then(function() {
-    document.getElementById("splash-screen").style.display = "block";
-    document.getElementById("background-screen").style.display = "none";
+    showSplashScreen();
```

Comment:
> Also, if !document.hidden, aren't we calling showSplashScreen twice? Once here and once in main.js?

Yes. And here we're also hiding the background screen, even though it may not ever have been shown. But it's ok, because those should be effectively noops.